Formula/scriptcs.rb

```diff
@@ -1,8 +1,8 @@
 class Scriptcs < Formula
   desc "Tools to write and execute C#"
   homepage "https://github.com/scriptcs/scriptcs"
-  url "https://github.com/scriptcs/scriptcs/archive/v0.17.0.tar.gz"
-  sha256 "3ddf5f782d2092b2c1dd8068b2fbd97dca653ca88ade2f3aa4d764e6f2f04318"
+  url "https://github.com/scriptcs/scriptcs/archive/v0.17.1.tar.gz"
+  sha256 "e876118d82f52cbdd9569783ec9278c4ac449055aa628cdcb2d785bf8098a434"
 
```

Comment:
Just `depends_on "mono"`. And can you please squash your commits into one? With commit message `scriptcs 0.17.1`, that would be perfect! Many thanks.